packages/barcode-scanning/src/definitions.ts

```diff
@@ -116,6 +116,14 @@ export interface BarcodeScannerPlugin {
    * @since 5.4.0
    */
   getMaxZoomRatio(): Promise<GetMaxZoomRatioResult>;
+  /**
+   * Set the camera's focus point using normalized coordinates.
+   *
+   * Only available on Android and iOS.
+   *
+   * @since 8.0.0
```

Comment:
```suggestion
   * @since 7.3.0
```

Please run `npm run docgen` again.